src/popup.js

```diff
@@ -245,6 +247,8 @@ const getCurrentTabOrigin = async () => {
     document.querySelector(".page-specific-options-wrapper").remove()
   }
 
+  versionString.textContent = devTool.isFirefox ? browser.runtime.getManifest().version : chrome.runtime.getManifest().version
```

Comment:
According to the MDN docs firefox uses `browser` instead of `chrome.` namespace and the `isFirefox` check was already there, thats why i decided to go this way.

Reply:
Yes you're right, `browser` is be preferred in Firefox. Thing is the Browser have the following compatibility:    

|  |  |
| --- | --- |
| **Safari** | [.chrome and .browser](https://developer.apple.com/documentation/safariservices/safari_web_extensions/assessing_your_safari_web_extension_s_browser_compatibility#3586646) |
| **Firefox** | [.chrome and .browser](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API#browser_api_differences) |
| **Chrome** | [.chrome](https://issues.chromium.org/issues/40556351) |

It would be nice to use `browser`, but as long as Chrome doesn't support `browser` and `chrome` is the only cross-browser compatible option, I would prefer to just use `chrome` for now.   
Hopefully we can switch in the future to a more generic namespace like `browser`

Reply:
changed, review again.